src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Popup/BasePopup.shared.cs

```diff
@@ -0,0 +1,176 @@
+		/// </summary>
+		public BasePopup()
+		{
+			Color = Color.White;
+			BorderColor = default(Color);
+			VerticalOptions = LayoutOptions.CenterAndExpand;
+			HorizontalOptions = LayoutOptions.CenterAndExpand;
+			IsLightDismissEnabled = true;
+		}
+
+		public static BindableProperty ColorProperty = BindableProperty.Create(nameof(Color), typeof(Color), typeof(BasePopup));
+		public static BindableProperty SizeProperty = BindableProperty.Create(nameof(Size), typeof(Size), typeof(BasePopup));
+
+		public static BindableProperty VerticalOptionsProperty = BindableProperty.Create(nameof(VerticalOptions), typeof(LayoutOptions), typeof(BasePopup), LayoutOptions.CenterAndExpand);
+		public static BindableProperty HorizontalOptionsProperty = BindableProperty.Create(nameof(HorizontalOptions), typeof(LayoutOptions), typeof(BasePopup), LayoutOptions.CenterAndExpand);
+
+		/// <summary>
+		/// Gets or sets the <see cref="View"/> content to render in the Popup.
+		/// </summary>
+		/// <remarks>
+		/// The View can be or type: <see cref="View"/>, <see cref="ContentPage"/> or <see cref="NavigationPage"/>
+		/// </remarks>
+		public virtual View Content { get; set; }
+
+		/// <summary>
+		/// Gets or sets the <see cref="Color"/> of the Popup.
+		/// </summary>
+		/// <remarks>
+		/// This color sets the native background color of the <see cref="Popup"/>, which is
+		/// independent of any background color configured in the actual View.
+		/// </remarks>
+		public Color Color
+		{
+			get => (Color)GetValue(ColorProperty);
+			set => SetValue(ColorProperty, value);
+		}
+
+		/// <summary>
+		/// Gets or sets the <see cref="LayoutOptions"/> for positioning the <see cref="Popup"/> vertically on the screen.
+		/// </summary>
+		public LayoutOptions VerticalOptions
+		{
+			get => (LayoutOptions)GetValue(VerticalOptionsProperty);
+			set => SetValue(VerticalOptionsProperty, value);
+		}
+
+		/// <summary>
+		/// Gets or sets the <see cref="LayoutOptions"/> for positioning the <see cref="Popup"/> horizontally on the screen.
+		/// </summary>
+		public LayoutOptions HorizontalOptions
+		{
+			get => (LayoutOptions)GetValue(HorizontalOptionsProperty);
+			set => SetValue(HorizontalOptionsProperty, value);
+		}
+
+		// REVIEW - 11/30/2020 - ahoefling
+		// In the original code review https://github.com/xamarin/Xamarin.Forms/pull/9616
+		// It was recommended to configure BorderColor as a platform specific attached
+		// property since it is only usable on UWP. 
+		//
+		// Do we want to leave this as part of the API with plans to implement or
+		// should we move it as an attached property.
+
+		/// <summary>
+		/// Gets or sets the <see cref="Color"/> of the Popup Border.
+		/// </summary>
+		/// <remarks>
+		/// This color sets the native border color of the <see cref="Popup"/>, which is
+		/// independent of any border color configured in the actual view.
+		/// </remarks>
+		public Color BorderColor { get; set; } // UWP ONLY - wasn't originally in spec
```

Comment:
In the original code review it was discussed this should be a platform specific attached property. I left it in so we could discuss if we still wanted to do that or if we wanted to support this API across the platforms.

Reply:
on XF 5.0 we have access to the OnPlatform API, so we can take advantage of that. To me makes sense to have

Reply:
I think that right now we can't expose in this way to our users, because they don't use Multi-Target project. Since this is a platform specific thing, you can use [this approach](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/platform/android/listview-fast-scrolling). The Forms opened this API for us. If you need any help implementing this approach, please let me know.

Reply:
That is exactly what I was looking for. I knew it was possible to do something like that in XF but wasn't sure how. I'll update this on the next code push to use that approach

Reply:
@ahoefling just be aware that we can do it the same [as XF](https://github.com/xamarin/Xamarin.Forms/tree/5.0.0/Xamarin.Forms.Core/PlatformConfiguration/WindowsSpecific).

Reply:
This has been fixed and will be updated in the next code push